doc/designs/evp-cipher-pipeline.md

```diff
@@ -0,0 +1,144 @@
+EVP APIs for supporting cipher pipelining in provided ciphers
```

Comment:
Can you break the long lines before pushing those changes? The line limit is ~80 characters so it can be also conveniently viewed in terminal.

Reply:
Fixed line length and other formatting. Changes ran through `util/check-format.pl`